quandl/util.py

```diff
@@ -51,3 +51,36 @@ def convert_to_dates(dic):
             elif isinstance(v, dict):
                 Util.convert_to_dates(v)
         return dic
+
+    @staticmethod
+    def convert_options(**options):
+        new_options = dict()
+        if 'params' in options.keys():
+            for key, value in options['params'].items():
+                is_dict = False
+                if isinstance(value, list):
+                    key = key + '[]'
+                else:
+                    if isinstance(value, dict):
+                        new_value = dict()
+                        is_dict = True
+                        old_key = key
+                        for k, v in value.items():
```

Comment:
Ran into an issue if `value` is an empty dictionary it would error out. I think if this is the case we should ignore and process the next value.